students/ZhuojingXie/lesson04/mailroom2.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+
+import sys
+import os
+
+data = [{'name':'Andy','donation':[960,256,123.5,40]},{'name':'Bryce','donation':[30,45,27]},
+        {'name':'Charile','donation': [25,50]},{'name':'David', 'donation':[10]},
+        {'name':'Elaine','donation':[75,26]}]
+
+def menu_selection(prompt, dispatch_dict):
+    while True:
+        response = input(prompt)
+        if dispatch_dict[response]() == 'exit menu':
+            break
```

Comment:
`sys.exit()` will exit the script, you technically don't need to do anything if the quit option is mapped to sys.exit